### ISSUE TITLE
WebXRManager throws error in onSessionEnd if getController is never called

### ISSUE THREAD

**Issue by kalegd** [mrdoob/three.js#23983]
TLDR: Lack of null check for controllers in `onSessionEnd` causes errors if `getController` is never called

It seems controllers are only created when `getController` or `getControllerGrip` are called. if those functions are never called, we end up with null controllers in the inputSourcesMap

https://github.com/mrdoob/three.js/blob/1238a324e081752d9f66219252361f05f73594df/src/renderers/webxr/WebXRManager.js#L337-L344

Once the session ends, there's no null check for the controllers when trying to disconnect them as seen below

https://github.com/mrdoob/three.js/blob/1238a324e081752d9f66219252361f05f73594df/src/renderers/webxr/WebXRManager.js#L132-L138
